### ISSUE TITLE
Correct syntax for targeting field inside block on CLI

### ISSUE THREAD

**Issue by onebrightlight** [spicywebau/craft-neo#664]
#### Description
If i'm trying to set a field i'm using in neo, how do i target it specifically with `./craft resave/neo-blocks` using `--set` and `--to`? i know it uses the same syntax as resave/matrix-blocks but i don't know specifically how i'm supposed to target the block/element first.

For context, i have added a custom field with [@trendyminds/craft-design-tokens](https://github.com/trendyminds/craft-design-tokens) and since this is being added after the fact, in many cases, the fields this needs to be used on are currently empty unless the values are changed and resaved.

- My Neo field’s handle is `contentBuilder`
- It exists on a block type of `textBlock`
- The field handle is `mediaStyle` and i'm trying to set it to a value of "Rounded Shadow" for my example, though it most likely is a full config i need to set.

i thought that i could target it correctly this way: `craft resave/neo-blocks --field contentBuilder --set mediaStyle --to "=Rounded Shadow" --if-empty` but i get the following stack trace error:

```bash
ddev exec php craft resave/neo-blocks --field contentBuilder --set mediaStyle --to "=Rounded Shadow" --if-empty
Resaving 391 Neo blocks ...
    - [1/391] Resaving 1149 (1149) ... Exception 'yii\base\UnknownPropertyException' with message 'Setting unknown property: benf\neo\elements\Block::mediaStyle'

in /var/www/html/vendor/yiisoft/yii2/base/Component.php:209

Stack trace:
#0 /var/www/html/vendor/craftcms/cms/src/base/Element.php(2006): yii\base\Component->__set('mediaStyle', 'Rounded')
#1 /var/www/html/vendor/craftcms/cms/src/console/controllers/ResaveController.php(439): craft\base\Element->__set('mediaStyle', 'Rounded')
#2 [internal function]: craft\console\controllers\ResaveController->craft\console\controllers\{closure}(Object(craft\events\BatchElementActionEvent))
#3 /var/www/html/vendor/yiisoft/yii2/base/Component.php(633): call_user_func(Object(Closure), Object(craft\events\BatchElementActionEvent))
#4 /var/www/html/vendor/craftcms/cms/src/services/Elements.php(993): yii\base\Component->trigger('beforeResaveEle...', Object(craft\events\BatchElementActionEvent))
#5 /var/www/html/vendor/craftcms/cms/src/console/controllers/ResaveController.php(462): craft\services\Elements->resaveElements(Object(benf\neo\elements\db\BlockQuery), true, true, false, false)
#6 /var/www/html/vendor/craftcms/cms/src/console/controllers/ResaveController.php(337): craft\console\controllers\ResaveController->_resaveElements(Object(benf\neo\elements\db\BlockQuery))
#7 /var/www/html/vendor/spicyweb/craft-neo/src/Plugin.php(219): craft\console\controllers\ResaveController->resaveElements('benf\\neo\\elemen...', Array)
#8 [internal function]: benf\neo\Plugin->benf\neo\{closure}('Shadow')
#9 /var/www/html/vendor/craftcms/cms/src/console/CallableAction.php(43): call_user_func_array(Object(Closure), Array)
#10 /var/www/html/vendor/yiisoft/yii2/base/Controller.php(178): craft\console\CallableAction->runWithParams(Array)
#11 /var/www/html/vendor/yiisoft/yii2/console/Controller.php(182): yii\base\Controller->runAction('neo-blocks', Array)
#12 /var/www/html/vendor/craftcms/cms/src/console/Controller.php(224): yii\console\Controller->runAction('neo-blocks', Array)
#13 /var/www/html/vendor/yiisoft/yii2/base/Module.php(552): craft\console\Controller->runAction('neo-blocks', Array)
#14 /var/www/html/vendor/yiisoft/yii2/console/Application.php(180): yii\base\Module->runAction('resave/neo-bloc...', Array)
#15 /var/www/html/vendor/craftcms/cms/src/console/Application.php(90): yii\console\Application->runAction('resave/neo-bloc...', Array)
#16 /var/www/html/vendor/yiisoft/yii2/console/Application.php(147): craft\console\Application->runAction('resave/neo-bloc...', Array)
#17 /var/www/html/vendor/craftcms/cms/src/console/Application.php(121): yii\console\Application->handleRequest(Object(craft\console\Request))
#18 /var/www/html/vendor/yiisoft/yii2/base/Application.php(384): craft\console\Application->handleRequest(Object(craft\console\Request))
#19 /var/www/html/craft(13): yii\base\Application->run()
#20 {main}
Failed to execute command php craft resave/neo-blocks --field contentBuilder --set mediaStyle --to =Rounded Shadow --if-empty: exit status 1
```

Basically i'm looking to figure out what, if anything is wrong with my syntax and what i need to change to make this work, as the specifics of targeting values in [Craft’s Console Commands for resaving Matrix Blocks](https://craftcms.com/docs/4.x/console-commands.html#resave-matrix-blocks) are not completely clear to me, either.

#### Other information
- Neo version: 3.4.0
- Craft version: 4.2.5.1

**Comment by onebrightlight** [comment]
Okay, thanks to @brianjhanson's assistance, i’ve sussed out that what i’m missing is `--type textBlock` from my command. Whoops. The one thing i still have to figure out is how to actually store the value from the Design Tokens plugin, which should be something like `{"config":"media-style.json","key":"rounded shadow"}` but that's not really relevant to this issue. :)
